.github/workflows/ci.yml

```
@@ -4,31 +4,36 @@ on:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Check out the repository
+      - name: Check out repository
         uses: actions/checkout@v4
 
-      - name: Determine label
-        id: determine_label
+      - name: Determine label based on target branch
+        id: determine-label
         run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:23: Double quote to prevent globbing and word splitting

(shellcheck)